Source/ISHRemote/Trisoft.ISHRemote/Connection/InfoShareWcfSoapWithOpenIdConnectConnection.cs

```diff
@@ -277,35 +262,23 @@ public InfoShareWcfSoapWithOpenIdConnectConnection(ILogger logger, HttpClient ht
             ResolveServiceUris();
 
 #if NET48
-            //extract from WcfSoapWithWsTrustConnection::ResolveEndpoints function to get to binding
             _logger.WriteDebug("InfoShareWcfSoapWithOpenIdConnectConnection Resolving Binding (NET48)");
-            Uri wsdlUriApplication = new Uri(InfoShareWSBaseUri, _serviceUriByServiceName[Application25] + "?wsdl");
-            var wsdlImporterApplication = GetWsdlImporter(wsdlUriApplication);
-            // Get endpont for http or https depending on the base uri passed
-            var applicationServiceEndpoint = wsdlImporterApplication.ImportAllEndpoints().Single(x => x.Address.Uri.Scheme == InfoShareWSBaseUri.Scheme);
-
+            _commonBinding = new WS2007FederationHttpBinding(WSFederationHttpSecurityMode.TransportWithMessageCredential);
+            _commonBinding.Security.Message.IssuedKeyType = SecurityKeyType.BearerKey;
+            _commonBinding.Security.Message.IssuedTokenType = "http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV2.0";
+            _commonBinding.Security.Message.EstablishSecurityContext = false;

```

Comment:
It looks like the pragma if-else-end NET48 can end here... the rest of the code below looks like an exact duplication